### PR TITLE
Improve owner/maintainer documentation

### PR DESCRIPTION
**Pull request by jMuzsik**
Hi, simple fix, only had to move some lines around/add text. Thanks!

### CODE REVIEW

File: warehouse/templates/manage/roles.html [hunk 1]
```diff
@@ -23,10 +23,10 @@ <h2>Collaborators</h2>
   <div class="callout-block">
     <p>There are two possible roles for collaborators:</p>
     <dl>
+      <dt>Maintainer</dt>
+      <dd>May upload releases for a project. Cannot add collaborators, delete files, delete releases or delete the project.</dd>
       <dt>Owner</dt>
       <dd>Owns a package, may add other collaborators for that package, and upload releases for a package.</dd>
```
Comment (di) [inline warehouse/templates/manage/roles.html]: While you're here, could you update this line to indicate that owners _can_ delete releases and _can_ delete the project?

**Comment by jMuzsik** [comment]
Ok, I edited the text a little. Replaced all may's with can's. Also, while reading closely I thought it to be preferable to reorder the order of ideas. Do you think it looks good?

One thing I am wondering is if I ought to delete the reference to `Owns a package` b/c maintainer has no reference to what he does. So perhaps I ought to write `Helps maintain a package.` or delete `Owns a package`?

**Comment by di** [comment]
@jMuzsik I think deleting the "Owns a package" is fine, it's a bit redundant anyways 🙂

**Comment by di** [comment]
Thanks @jMuzsik!

**Pull request merged**